content/blog/2020-03-19-howto/index.en.md

```diff
@@ -0,0 +1,81 @@
+
+
+# 1. Article creation
+
+**Étape 1.1.** Enregister, sous le nom `index.en.md`/`index.fr.md` (anglais/français), le fichier .md [source](https://github.com/dot-layer/blog/blob/master/content/blog/2020-03-19-howto/index.fr.md) utilisé pour créer le présent article. Cette étape sert simplement à vous éviter de copier l'en-tête (en format [YAML](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html)), qui contient quelques champs essentiels à remplir.
+
+**Step 1.1.** Save, under the name `index.en.md` /` index.fr.md` (English / French), the .md file [source](https://github.com/dot-layer/blog/blob/master/content/blog/2020-03-19-howto/index.en.md) used to create this article. This step is simply to save you from copying the header (in [YAML] format(https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html)), which contains some essential fields to fill in.
+
+**Step 1.2.** The YAML header for this article is as follows.
+
+```
+---
+title: How to submit an article
+author: Samuel Perreault and David Beauchemin
+date: '2020-04-21'
+slug: howto-en
+type: post
+categories: ["Contribute"]
+tags: []
+description: "A template to contribute to the blog .Layer"
+featured: "howto-cover.png"
+featuredpath: "img/headers/"
+---
+```
+The fields are almost all *self-explanatory*.
+The `date` field should contain the creation date of the file. We will change it for the publication date in due time (in the *Pull Request*).
+We will also take care of the `categories` and` tags` fields.
+The `slug` field is a *nickname* for your article which will be used to name the various files linked to the article in the blog directory.
+The `featured` field should contain the name of the cover image file, while` featuredpath` (which should remain unchanged) indicates where to find the file. This is where you should place your cover image.
+Finally, if the article submitted is already published on another platform, please add the `canonical` field to specify the platform. For example, in the header of the article *What's wrong with Scikit-Learn* published on the blog, we find
+```
+canonical: https://www.neuraxio.com/en/blog/scikit-learn/2020/01/03/what-is-wrong-with-scikit-learn.html
+```
+in addition to the mention at the end of the article which refers to the original publication.
+
+
+As for writing the article, you have to know the basics of Markdown.
+In addition to the content already in the *template* that constitutes this article, we recommend this little [*cheatsheet*] (https://github.com/adam-p/markdown-here/wiki/Markdown-Here-Cheatsheet) to use Markdown.
+
+# 2. Submission of an article
+
+**Steps 2.1. and 2.2.** These are classic operations of [Git](https://git-scm.com/). If you are not comfortable with Git, let us know and we will make it up! Otherwise, Atlassian offers a great free online [training](https://www.atlassian.com/fr/git).
```

Comment:
```suggestion
**Steps 2.1. and 2.2.** These are classic operations of [Git](https://git-scm.com/). If you are not comfortable with Git, let us know and we will make it up! Otherwise, Atlassian offers great free online [training](https://www.atlassian.com/fr/git).
```